### PR TITLE
upgrade to asp.net core final

### PR DESCRIPTION
**Pull request by Wyntuition**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
   "repository": "https://github.com/jincod/AspNet5DemoApp",
-  "keywords": ["aspnet5", "vnext"],
+  "keywords": ["aspnetcore", "vnext"],
   "buildpacks": [
     {
       "url": "https://github.com/heroku/heroku-buildpack-nodejs"
     },
     {
-      "url": "https://github.com/jincod/dotnet-buildpack"
+      "url": "https://github.com/noliar/dotnet-buildpack"
```
Comment (jincod) [inline app.json:11]: Revert back url to jincod

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "description": "Demp app",
   "repository": {
     "type": "git",
-    "url": "https://github.com/jincod/AspNet5DemoApp.git"
+    "url": "https://github.com/Wyntuition/AspNet5DemoApp"
```
Comment (jincod) [inline package.json:8]: Revert back url to Jincod

**Comment by Wyntuition** [comment]
I don't think it will work unless that buildback is updated for the .net core final release.

**Comment by jincod** [comment]
Hi @Wyntuition 

I have created a new dotnetcore-buildback https://github.com/jincod/dotnetcore-buildpack. Also I have updated this project to ASP.NET Core Preview and deployed demo version to https://aspnet5demoapp.herokuapp.com/.

Could you please test this buildback on your projects?

**Comment by jincod** [comment]
https://blogs.msdn.microsoft.com/dotnet/2017/01/30/announcing-net-core-net-native-and-nuget-updates-in-vs-2017-rc/#projectjsonxproj---csproj-migration

**Pull request closed without merging**